Dockerfile

```diff
@@ -17,8 +17,8 @@ COPY . .
 
 ENV SC_THEME=default
 ENV TZ=Europe/Berlin
+RUN gulp clear-cache
```

Comment:
könnte man dass dann nicht direkt in den gulp standard task bauen? Der wird ja nur hier verwendet.